Assets/Scripts/Player/PlayerController.cs

```diff
@@ -16,37 +12,37 @@ public class PlayerController : MonoBehaviour {
         this.floorMask = LayerMask.GetMask ("Floor");
     }
 
-    void Start () {
-        this.playerRigidbody = GetComponent<Rigidbody> ();
+    protected override void Start () {
+        base.Start();
         this.inventory = GetComponent<PlayerInventory> ();
+        baseStats [Stat.Speed] = 5.0f;
```

Comment:
Okay. Yeah, I was wrestling with how to get these things visible in the inspector. I thought the problem was being caused by inheritance, but maybe it was the use of a Dictionary. I had to read your suggestion several times, but I think I understand what you're saying. If there is a class member for each stat then the values can be manipulated in the inspector and saved into prefabs. Then at Start() all those values can be pulled into a collection so that they can be used in the effectiveStats calculation that happens each frame. Right? I think that's what you're suggesting...